citest/google_session/__init__.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""Citest support modules for automating Google sessions and authenticated requests."""
+"""Citest support modules for automating Google sessions and authenticated
+requests.
```

Comment:
First line comment acts as a summary, so dont break it like this. Reword it instead so summary itself is 80 cols.

What does "automated" mean? What is a "Google session"? If there is some GCP concept around this, then provide a URL to GCP documentation that discusses it. If this is a spinnaker thing, it should be under spinnaker_testing. If it is common to GCP, should it be under gcp_testing?